scripts/packages_exporter.py

```diff
@@ -268,18 +271,38 @@ async def make_request(
         method: str,
         endpoint: str,
         params: dict = None,
-        data: dict = None,
+        body: dict = None,
+        user_headers: dict = None,
+        data: typing.Optional[list] = None,
+        send_to: typing.Literal['web_server', 'sign_server'] = 'web_server',
     ):
-        full_url = urllib.parse.urljoin(settings.sign_server_url, endpoint)
+        if send_to == 'web_server':
+            headers = {**self.web_server_headers}
+            full_url = urllib.parse.urljoin(settings.albs_api_url, endpoint)
+        elif send_to == 'sign_server':
+            headers = {}
+            full_url = urllib.parse.urljoin(
+                settings.sign_server_api_url, endpoint
+            )
+        else:
+            raise ValueError(
+                "send_to parameter must be web_server of sign_server"
+            )
+
+        if user_headers:
+            headers.update(user_headers)
+
         async with aiohttp.ClientSession(
-            headers=self.headers, raise_for_status=True
+            headers=headers, raise_for_status=True
         ) as session:
             async with session.request(
-                method, full_url, json=data, params=params
+                method, full_url, json=body, params=params, data=data
             ) as response:
-                json_data = await response.read()
-                json_data = json.loads(json_data)
-                return json_data
+                if response.headers['Content-Type'] == 'application/json':
+                    json_data = await response.read()
+                    json_data = json.loads(json_data)
```

Comment:
you can use `await response.json()` instead

Reply:
done